scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -444,6 +444,9 @@ class FossId internal constructor(
     ): String {
         // we ignore the revision because we want to do a delta scan
         val recentScans = scans.recentScansForRepository(url)
+
+        log.info { "${recentScans.size} recent scans have been found." }
```

Comment:
Consider (optional): "Found ${recentScans.size} scans."

It's a not shorter and doesn't start the sentence with a number.

Reply:
Oh just read @sschuberth's proposal to strip credentials. +1 for that instead